README.md

```diff
@@ -170,51 +182,47 @@ A number of callback functions are fired at various points of the flow. The most
     containerId: 'onfido-mount',
     // here we send the data in the complete callback
     onComplete: function() {
-      var captures = Onfido.getCaptures()
-      sendToServer(captures)
+      //communicate with your backend and initiate the check
```

Comment:
This still implies you could create a check from the client. What about `Tell your backend service that it can create the check`